### PR TITLE
Document release process on contribute page

### PR DESCRIPTION
**Pull request by ann0see**
Merges https://github.com/jamulussoftware/jamulus/pull/1594 and https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md with some minor updates
# Does this need translation?

- [ ] Yes 
- [x] No

### CODE REVIEW

**Review by pljones — APPROVED** [review]
Looks good.

**Comment by ann0see** [comment]
Thanks!

Maybe we should post a link to this file in the RELEASE-PROCESS.md file on the main repo.

**Comment by pljones** [comment]
Probably a good idea, yes.

File: contribute/en/Release-Process.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+~~~
+---
+name: 'For maintainers: Release a new version'
+about: This is the Developer-only template which we use for tracking the process of future releases.
+title: 'Prepare Release 3.y.z'
+labels: release
+assignees: ''
+---
+
```
Comment (hoffie) [inline contribute/en/Release-Process.md:165]: @ann0see I think this part is not necessary on the website. I added it while the plan was still to make it an actual issue template.

**Pull request merged**